### PR TITLE
Add versioning and tagging

### PR DESCRIPTION
**Pull request by plorenz**

### CODE REVIEW

File: ziti/sdkinfo/host.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*
+	Copyright 2019 NetFoundry, Inc.
```
Comment (dovholuknf) [inline ziti/sdkinfo/host.go]: time machine! :)

File: ziti/sdkinfo/host_unix.go [hunk 2]
```diff
@@ -0,0 +1,32 @@
+// +build !windows
+
+/*
+	Copyright 2019 NetFoundry, Inc.
```
Comment (dovholuknf) [inline ziti/sdkinfo/host_unix.go]: 2020 - there's a few other spots to update from 2019

File: ziti/sdkinfo/host.go (hunk 1 above)
Comment (plorenz) [reply ziti/sdkinfo/host.go]: fixed

File: ziti/sdkinfo/host_unix.go (hunk 2 above)
Comment (plorenz) [reply ziti/sdkinfo/host_unix.go]: fixed

**Pull request merged**